### ISSUE TITLE
Event Detail page

### ISSUE THREAD

**Issue by schalkneethling** [mdn/developer-portal#1736]
1. [ ] "Upcoming sessions:" should be a heading element

## Related items

1. [x] Related items heading should be an `h2` element
2. [x] Items list should be marked up as a list and headings should be `h3` elements:

```
<ul class="mzp-l-card-quarter">
  <li class="class="mzp-c-card mzp-c-card-extra-small mzp-has-aspect-16-9"">
    <a href="/videos/compiler-compiler-part-1-fixing-a-bug-834b676b6b25/" class="mzp-c-card-block-link" data-type="video">
        <h3 class="mzp-c-card-title">
          Compiler Compiler - Part 1: Fixing a bug
        </h3>
    </a>
  </li>
  ...
</ul>
```

**Comment by stevejalim** [comment]
Note to self: check whether this is already handled by other a11y improvements

**Comment by stevejalim** [comment]
@valgrimm This item that Schalk spotted 

>  "Upcoming sessions:" should be a heading element

is a content one on https://mozilla.dev/events/compiler-compiler-yulia-startsev/

**Comment by valgrimm** [comment]
fixed

**Comment by stevejalim** [comment]
Reopened because there is a PR attached to this that I hope wasn't auto closed by GitHub
